www/src/components/PropsTable.tsx

```diff
@@ -1,11 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { MDXRenderer } from 'gatsby-plugin-mdx';
-import { Badge, Card } from '~paragon-react';
+import { Badge, Card, Hyperlink } from '~paragon-react';
 import PropType from './PropType';
 
 const IGNORED_COMPONENT_PROPS = ['intl'];
 
+const bootstrapLinks = {
+  Button: 'https://react-bootstrap-v4.netlify.app/components/buttons/#button-props',
```

Comment:
Can we extract `https://react-bootstrap-v4.netlify.app/components` into a constant? This might be useful in the future if we upgrade to react-bootstrap v5 and the URL will change

Reply:
Corrected, thanks